src/Filter.php

```diff
@@ -338,7 +338,11 @@ public function validate($modelClass)
      */
     public static function parseFromParameters($parameters, $modelClass)
     {
-        if (!isset($parameters->filter)) {
+        if (
+            !isset($parameters->filter) ||
+            empty($parameters->filter) || // empty strings or arrays
+            (!is_array($parameters->filter) && !is_object($parameters->filter)) // should allow arrays and objects
```

Comment:
if we had PHP  >= 7.1 we could use https://www.php.net/manual/en/function.is-iterable.php instead

Reply:
`is_iterable` doesn't work on objects, from the docs `var_dump(is_iterable(new stdClass()));  // bool(false)`

Reply:
We might throw `IncorrectParametersException` when`!is_array($parameters->filter)` to warn them/us. What do you think?

Reply:
Might be better than to ignore it, since the clients would expect it to work